src/AdvApi32/AdvApi32+ServiceType.cs

```diff
@@ -59,7 +59,12 @@ public enum ServiceType : uint
             /// <summary>
             /// The service can interact with the desktop.
             /// </summary>
-            SERVICE_INTERACTIVE_PROCESS = 0x00000100
+            SERVICE_INTERACTIVE_PROCESS = 0x00000100,
+            
```

Comment:
I think that it's this line that contains whitespace that is breaking the CI build for this PR